config.lua

```diff
@@ -10,6 +10,9 @@ travelnet.travelnet_sound_enabled  = minetest.settings:get_bool("travelnet.trave
 -- if you set this to false, travelnets cannot be created
 -- (this may be useful if you want nothing but the elevators on your server)
 travelnet.travelnet_enabled        = minetest.settings:get_bool("travelnet.travelnet_enabled", true)
+
+travelnet.travelnet_migrate_meta = minetest.settings:get_bool("travelnet.travelnet_migrate_meta", false)
```

Comment:
Should this be renamed as it is not really enabling/disabling migration or affecting migration itself in any way but instead just changing how migration is done?:
Enables LBM for cleanup instead of waiting for player to interact with node to do cleanup.

Like `travelnet.travelnet_cleanup_lbm` / `travelnet.travelnet_cleanup_on_map_load` or something like that.

Reply:
renamed to `travelnet.travelnet_cleanup_lbm`

> Shouldn't we use this flag to disable the old migration behaviour too?

if you mean the "restore_network_via_abm" thing: yeah, sure but i would prefer that in another PR if possible

Reply:
> if you mean the "restore_network_via_abm" thing: yeah, sure but i would prefer that in another PR if possible

I believe it was about disabling this if lbm is enabled (if doing that then maybe also just set empty if config option is disabled / false):
https://github.com/mt-mods/travelnet/blob/65c254a37273fcc588d51e1fcb94316866f5c5a7/register_travelnet.lua#L16-L19

Reply:
Yeah that's what I meant